mod-circulation/src/main/resources/vega/mod-circulation/features/util/initData.feature

```diff
@@ -357,3 +357,11 @@ Feature: init data for mod-circulation
     And request updateAccountRequest
     When method PUT
     Then status 204
+
+   @EnableTlrFeature
+   Scenario: enable title level request
+    * def enableTlrRequest = read('classpath:vega/mod-circulation/features/samples/enable-tlr-config-entity-request.json')
+    Given path 'configurations/entries'
+    And request enableTlrRequest
+    When method POST
+    Then status 201
```

Comment:
this may not return 201 if TLR is enabled, have you run your test several times?

Reply:
Yes, I have. I think every time created data is cleaned after tests are stopped.